packages/block-library/src/list-item/edit.js

```diff
@@ -32,8 +34,24 @@ import {
 import { convertToListItems } from './utils';
 
 export function IndentUI( { clientId } ) {
-	const [ canIndent, indentListItem ] = useIndentListItem( clientId );
-	const [ canOutdent, outdentListItem ] = useOutdentListItem( clientId );
+	const indentListItem = useIndentListItem( clientId );
+	const outdentListItem = useOutdentListItem( clientId );
+	const canIndent = useSelect(
+		( select ) => select( blockEditorStore ).getBlockIndex( clientId ) > 0,
+		[ clientId ]
+	);
+	const canOutdent = useSelect(
+		( select ) => {
+			const { getBlockRootClientId, getBlockName } =
+				select( blockEditorStore );
+			return (
+				getBlockName(
+					getBlockRootClientId( getBlockRootClientId( clientId ) )
+				) === 'core/list-item'
+			);
+		},
+		[ clientId ]
```

Comment:
Wy not merge these two useSelect calls?

Reply:
Good point. I'm so focussed on block render that I forgot about the controls :D